src/abc/Module.js

```diff
@@ -1,6 +1,10 @@
 export default class Module {
     constructor(app, name) {
     	this.App = app;
+        this.Name = name;
+        this.Config = app?.config;
```

Comment:
@titkov49 Why do you need to define Config, Navigation and Router in here? Also, I think it should be without `?`. Also, be patient with the indentation on line 6 and 7